lib/components/src/controls/Array.tsx

```diff
@@ -12,6 +12,14 @@ const format = (value: ArrayValue, separator: string) => {
 
 export type ArrayProps = ControlProps<ArrayValue> & ArrayConfig;
 export const ArrayControl: FC<ArrayProps> = ({ name, value, onChange, separator = ',' }) => {
+  const text = format(value, separator);
```

Comment:
I don't think this is correct. The text value for Array is something like: `a,b,c`, not an actual JSON string.